tests/providers/google/cloud/operators/test_cloud_storage_transfer_service.py

```diff
@@ -301,6 +318,31 @@ def test_job_create_aws(self, aws_hook, mock_hook):
 
         assert result == VALID_TRANSFER_JOB_AWS
 
+    @mock.patch(
+        "airflow.providers.google.cloud.operators.cloud_storage_transfer_service.CloudDataTransferServiceHook"
+    )
+    def test_job_create_aws_with_role_arn(self, aws_hook, mock_hook):
```

Comment:
```suggestion
    @mock.patch(
        "airflow.providers.google.cloud.operators.cloud_storage_transfer_service.CloudDataTransferServiceHook"
    )
    @mock.patch("airflow.providers.google.cloud.operators.cloud_storage_transfer_service.AwsBaseHook")
    def test_job_create_aws_with_role_arn(self, aws_hook, mock_hook):
```

You've forgot to mock AwsBaseHook